runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -189,7 +180,10 @@ impl Bank {
                 .iter()
                 .filter_map(|account| account.as_ref())
                 .enumerate()
-                .map(|(i, account)| (&vote_account_rewards.rewards[i].0, account))
+                .map(|(i, account)| {
+                    total_rewards += vote_account_rewards.rewards[i].1.lamports;
+                    (&vote_account_rewards.rewards[i].0, account)
+                })
```

Comment:
```suggestion
                .inspect(|(i, _account)| {
                    total_rewards += vote_account_rewards.rewards[i].1.lamports;
                })
                .map(|(i, account)| {
                    (&vote_account_rewards.rewards[i].0, account)
                })
```

nit: can you move this side effect out of `map` and do it in `inspect` instead?